Gemfile

```diff
@@ -2,6 +2,14 @@ source 'https://rubygems.org'
 
 gemspec
 
+rails_version = ENV['RAILS_VERSION']
+case rails_version
+when /^(\d+\.\d+)/
+  gem 'rails', "~> #{$1}.0"
+else
+  gem 'rails'
```

Comment:
@Khalasar I've just seen that because of the dep (unpinned version of Rails) in the gemspec we can remove this `else` branch altogether.